git2-curl/Cargo.toml

```diff
@@ -16,7 +16,7 @@ edition = "2018"
 curl = "0.4.33"
 url = "2.0"
 log = "0.4"
-git2 = { path = "..", version = "0.17", default-features = false }
+git2 = { path = "..", version = "0.18", default-features = false }
```

Comment:
I think the version of git2-curl itself also needs to be updated.

Reply:
Updated!

Reply:
Hm, perhaps I'm not seeing it (or wasn't clear)? The version of `git2-curl` should probably be bumped from 0.18.0 to 0.19.0.